src/components/menu-right/DotTypeEditor.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="my-2">
+    <p data-test="menu-right--dot-type">Dot Type {{ dotTypeIndex }}</p>
```

Comment:
In this template, should we have an option to change dot appearance? (filled, outline, forward slash, backward slash, etc.)

Reply:
Yes, this needs to be done in a future issue. https://github.com/calband/calchart-redesign/issues/86